packages/react/src/primitives/Text/__test__/Text.test.tsx

```diff
@@ -1,8 +1,10 @@
-import { Text } from '../Text';
+import * as React from 'react';
+import { kebabCase } from 'lodash';
```

Comment:
Should we change this to `import kebabCase from 'lodash/kebabCase'`? Following this PR: #682 I know this isn't part of the change, but something we should track.

Reply:
I can do this as a separate PR